gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
The goal of this patch is to remove the environment variables used to use command line arguments instead. Let's just do this for now, and not something else - it is easy to do, and easy to review. Let's talk on irc about that!